src/devices/MatrixKeyboard/MatrixKeyboard.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Device.Gpio;
+using System.Linq;
+using System.Timers;
+
+namespace Iot.Device.MatrixKeyboard
+{
+    /// <summary>
+    /// GPIO Matrix Keyboard Driver
+    /// </summary>
+    public class MatrixKeyboard : IDisposable
+    {
+        /// <summary>
+        /// Get row pins
+        /// </summary>
+        public IEnumerable<int> RowPins => rowPins;
```

Comment:
I'm not sure I understand what RowPins and ColPins are supposed to represent. Is every row+column combo a key?

Reply:
Yes. 4 rows and 3 columns means there are 12 keys.

Reply:
please add a picture explaining how to connect this (unless it's labeled then just generic pic is fine)